queries/scala/highlights.scm

```diff
@@ -7,6 +7,15 @@
 ((identifier) @variable.builtin
  (#match? @variable.builtin "^this$"))
 
+; Assume other uppercase names constants.
+; NOTE: In order to distinguish constants we highlight
+; all the identifiers that are uppercased. But this solution
+; is not suitable for all occurences e.g. it will highlight
+; an uppercased method as a constant if used with no params.
+; Introducing highlighting for those specifi cases, is probably
+; best way to resolve the issue.
+((identifier) @constant (#match? @constant "^[A-Z]"))
```

Comment:
So in Scala you have constants identifiers like `Pi`, `MaxVertices`? Or rather `PI` , `MAX_VERTICES`

Reply:
`Pi` yes, check the [offical docs ](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables).